### PR TITLE
Make it clearer that the "greedy" option can drastically reduce planning time

### PR DESCRIPTION
**Pull request by renetapopova**
From 4.3 onwards, the `idp` option is the default.

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc [hunk 1]
```diff
@@ -176,12 +176,14 @@ This table describes the available query options for the connect-components plan
...
+[NOTE]
+Using this option can significantly _reduce_ the planning time of the query.
+| {check-mark}
 
 | connectComponentsPlanner=idp
 | Use the cost based IDP search algorithm when combining sub-plans.
-
-Note that using this option can significantly increase the planning time of the query.
+[NOTE]
+Using this option can significantly _increase_ the planning time of the query but usually finds better plans.
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc]:
```suggestion
[NOTE]
====
Using this option can significantly _increase_ the planning time of the query but usually finds better plans.
====
```

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc [hunk 2]
```diff
@@ -176,12 +176,14 @@ This table describes the available query options for the connect-components plan
 
 | connectComponentsPlanner=greedy
 | Use a greedy approach when combining sub-plans.
-| X
+[NOTE]
+Using this option can significantly _reduce_ the planning time of the query.
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc]:
```suggestion
[NOTE]
====
Using this option can significantly _reduce_ the planning time of the query.
====
```

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc [hunk 3]
```diff
@@ -69,7 +69,7 @@ Here we detail the available versions:
 | Query option | Description | Default
 | `3.5` | This will force the query to use Neo4j Cypher 3.5. |
 | `4.1` | This will force the query to use Neo4j Cypher 4.1. |
-| `4.2` | This will force the query to use Neo4j Cypher 4.2. As this is the default version, it is not necessary to use this option explicitly. | X
+| `4.2` | This will force the query to use Neo4j Cypher 4.2. As this is the default version, it is not necessary to use this option explicitly. | {check-mark}
```
Comment (martin-neotech) [inline cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc:78]:
```
//Check Mark
:check-mark: icon:check[]

//Cross Mark
:cross-mark: icon:times[]
```

Add this at line 3, see https://github.com/neo4j/neo4j-documentation/commit/1337e41225a2c9c92b6721b103144fb1dc0f3a37

**Review by martin-neotech — CHANGES_REQUESTED** [review]
Thanks, the check-mark needs to be defined at the top of the page.

File: cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc]: When I use this formating, the note doesn't render at all.

**Review by martin-neotech — APPROVED** [review]
Thanks

**Pull request merged**